src/chocolatey/infrastructure.app/configuration/ChocolateyConfiguration.cs

```diff
@@ -624,6 +624,7 @@ public ListCommandConfiguration()
         public bool ByIdOnly { get; set; }
         public bool ByTagOnly { get; set; }
         public bool IdStartsWith { get; set; }
+        public bool IgnorePinned { get; set; }

```

Comment:
We spoke about earlier today that we would try do avoid double negatives for the Configuration file itself.

Since this is a new option, would it maybe make sense to flip this to something like `ListPinnedPackage` and default it to `true` instead?

Reply:
Since we are already using `IgnorePinned` elsewhere, I am going to suggest that we stick with it for now, and look to rename everywhere, under a breaking change release.

Reply:
We already discussed this internally, and I agree to keep it so we are consistent.

It may however be a point to change it everywhere in the future.